lib/screens/player_screen.dart

```diff
@@ -174,17 +199,34 @@ class _PlayerScreenContent extends StatelessWidget {
                   return Column(
                     mainAxisAlignment: MainAxisAlignment.spaceAround,
                     children: [
-                      SizedBox(
-                          height: min(
-                              constraints.maxHeight -
-                                  controller.getTarget().height,
-                              constraints.maxWidth),
-                          width: constraints.maxWidth,
-                          child: const PlayerScreenAlbumImage()),
+                      SimpleGestureDetector(
+                        //TODO replace with PageView, this is just a placeholder
+                        onHorizontalSwipe: (direction) {
+                          final queueService = GetIt.instance<QueueService>();
+                          if (direction == SwipeDirection.left) {
+                            if (!FinampSettingsHelper
+                                .finampSettings.disableGesture) {
+                              queueService.skipByOffset(1);
+                            }
+                          } else if (direction == SwipeDirection.right) {
+                            if (!FinampSettingsHelper
+                                .finampSettings.disableGesture) {
+                              queueService.skipByOffset(-1);
+                            }
+                          }
+                        },
```

Comment:
You should add a tap gesture to toggle playback as well.

Reply:
As I mentioned in the other thread, I want this to be animated in the future. Do you have an idea how we could animate play/pause so that it's obvious what happend because of the tap?  
If not, I'd rather not add the gesture for now.

But I also remember someone requesting the swipe gesture, I'll see if they also requested play/pause :)

Reply:
I feel like playing/pausing is itself pretty obvious and easy to connect without an animation, and you don't have gestures for skipping yet either.  I guess you could just sort of flash a low opacity play/pause Icon over the cover like youtube does?

Reply:
yeah, I was thinking something similar. I guess that's good enough, I'll add the gesture :)